src/migrations/00041-validation-checks.sql

```diff
@@ -0,0 +1,23 @@
+ALTER TABLE "release tag"
+	-- It is necessary that each release tag has a tag key that has a Length (Type) that is greater than 0.
+	ADD CONSTRAINT "release tag$vGZu47lKJepQVH+hgSZNuUPdet2cG96akz3Yc8hta3A=" CHECK (0 < LENGTH("tag key")
+	AND LENGTH("tag key") IS NOT NULL
+	AND "tag key" = "tag key"
```

Comment:
I guess that this isn't wrong but it's weird seeing it here anyway.

Reply:
Yeah that should be easy enough to opimize out, not sure offhand why it's even included tbh